polyfill/lib/date.mjs

```diff
@@ -329,6 +336,5 @@ export class Date {
     return ES.CalendarCompare(GetSlot(one, CALENDAR), GetSlot(two, CALENDAR));
   }
 }
-Date.prototype.toJSON = Date.prototype.toString;
```

Comment:
I was wondering why this was necessary but I guess it's because you could otherwise say `Temporal.Date.prototype.toString.mydata = 3` and have it show up on `toJSON`?

Reply:
Indeed, or just running `Temporal.Date.prototype.toString === Temporal.Date.prototype.toJSON`